documentation/working-with-testcafe-studio/writing-test-code.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: docs
+title: Writing Test Code
+permalink: /documentation/working-with-testcafe-studio/writing-test-code
+---
+# Writing Test Code
+
+TestCafe Studio allows you to write tests using JavaScript and TypeScript. It provides a powerful **Code Editor** with syntax highlight, code completion, parameter hints and so on.
+
+To write test code from scratch, follow the steps:
+
+1. Create a `.js` or `.ts` file in a [test directory](organizing-tests.md#test-directory) or subdirectory. To do this, right-click the directory in the **Explorer** panel and select **New File** from the context menu. In the **New File** dialog, specify the file name and click **Create**.
+
+     IMAGE
+
+2. Double-click the test file in the **Explorer** panel to open it in the **Code Editor**.
+
+3. Write test code. For information about the API used in tests, see the [Test API](https://devexpress.github.io/testcafe/documentation/test-api/) topic in the open-source TestCafe documentation.
+
+## Converting Recorded Tests to JavaScript Code
+
+You can convert recorded tests to JavaScript code. This may be useful, for example, if you need to perform some specific actions which are easier to do with code.
```

Comment:
For me this explanation doesn't look quite clear

Reply:
Probably: "This may be useful, if you want to get more control and flexibility in your test."